src/include/com/amazonaws/kinesis/video/webrtcclient/NullableDefs.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Header file to define nullable values for common data types
+ */
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Public headers
+////////////////////////////////////////////////////
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
+#pragma clang diagnostic pop
+
+
+
+#define SET_DEFAULT_AS_NULL(a) \
```

Comment:
Due to no namespaces in C, I recommend prefixing these definitions with something like NULLABLE_

NULLABLE_SET_EMPTY()

and 

NULLABLE_CHECK_EMPTY()

Reply:
Agreed. Done